src/packages/engine-core/src/NodeEngine.ts

```diff
@@ -970,11 +965,7 @@ ${this.lastErrorLog.fields.file}:${this.lastErrorLog.fields.line}:${this.lastErr
   async version() {
     const prismaPath = await this.getPrismaPath()
 
-    const result = await execa(prismaPath, ['--version'], {
-      env: {
-        ...process.env,
-      },
```

Comment:
Why removing the env? If using custom binaries via env var I guess it's needed?

Reply:
Because passing in process.env is already the default. This is a no-op.